### PR TITLE
Add GitHub Actions workflows to migrate away from Travis-CI

### PR DESCRIPTION
**Pull request by alexhraber**
https://zendesk.atlassian.net/browse/EP-3955

travis going away, please review and merge, thanks.

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: repo-checks
+
+on:
+  pull_request:
+    branches:
+    - master
+    - deploy
```
Comment (grosser) [inline .github/workflows/actions.yml]: master should be enough

File: .github/workflows/actions.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        gem install activesupport -v 4.2.11.1 
+        gem install builder -v 3.2.3
+        gem install crass -v 1.0.5 
+        gem install loofah -v 2.3.1
+        gem install rails-html-sanitizer -v 1.3.0
+        gem install actionview -v 4.2.11.1
+        gem install rack -v 1.6.11
+        gem install actionpack -v 4.2.11.1 
+        gem install bump -v 0.8.0
+        gem install wwtd -v 1.4.0
```
Comment (grosser) [inline .github/workflows/actions.yml]: no need for that, `bundle install` should take care of that

**Review by grosser — APPROVED** [review]

File: .github/workflows/actions.yml (hunk 1 above)
Comment (everops-john) [reply .github/workflows/actions.yml]: okay

File: .github/workflows/actions.yml (hunk 2 above)
Comment (everops-john) [reply .github/workflows/actions.yml]: it was not finding these gems with bundle install. let's see if bundle update would do it.

**Pull request merged**